### ISSUE TITLE
WD1.1 - What We Do page template

### ISSUE THREAD

**Issue by sam-wmca** [wmcadigital/wmca-sprints#49]
**Epic:** See what WMCA does at a high level

***
#### As a
Resident/Private Sector/Public Sector/Media

#### I need to
Understand what WMCA does at a high level

#### So that
I know what the WMCA does

***
### Acceptance criteria
- [ ] Have a text/content area with a high level explanation (in Plain English)
- [ ] Links through to directorate hubs
- [ ] Test designs with users

***

**Comment by rolorogan** [comment]
# Update: 27th April

## What we do page updates:
### Front-end
- Add check to see if the hero image exists
- Link to the Hub-Hero partial view
- Add a check to see of the page text exists
- Add the page text into the template
- Add responsive grid classes from the [grid examples](https://deploy-preview-129--wmcads.netlify.app/styles/examples/grid-examples.html)
- Remove any custom styles not needed

### CMS
- Increase content card image size so it works responsively.

#### Commits: 7cb555f, 6aa339c, e57004e
